persistence/src/commonMain/kotlin/com/wire/kalium/persistence/db/UserDatabaseProvider.kt

```diff
@@ -1,15 +1,13 @@
 package com.wire.kalium.persistence.db
 
-import com.wire.kalium.persistence.dao.ConversationDAO
-import com.wire.kalium.persistence.dao.MetadataDAO
-import com.wire.kalium.persistence.dao.TeamDAO
-import com.wire.kalium.persistence.dao.UserDAO
+import com.wire.kalium.persistence.dao.*
 import com.wire.kalium.persistence.dao.asset.AssetDAO
 import com.wire.kalium.persistence.dao.client.ClientDAO
 import com.wire.kalium.persistence.dao.message.MessageDAO
 
 expect class UserDatabaseProvider {
```

Comment:
With every change to this KMP classes, we should implement / modify also in each platform, ie: `js`, `ios`, `android`.

So I think we still have to add the changes to `js` and `ios` targets 👀  If we currently don't have an implementation for that, we can add a dummy one see more at : `persistence/src/jsMain/kotlin/com/wire/kalium/persistence/db/UserDatabaseProvider.kt`

Reply:
Updated ios and js providers